recipe/tiledb-patches/system-ports/curl/vcpkg.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "curl",
```

Comment:
Just to clarify: Does TileDB depend on curl or libcurl? In other words, I was expecting a vcpkg system-port for libcurl, not curl.

Reply:
The `curl` vcpkg port provides both the library and optionally the tool.